stacks/api/function/runtime/node/entrypoint.ts

```diff
@@ -63,7 +66,35 @@ if (!process.env.EVENT_ID) {
       callArguments[0] = new Change(change);
       break;
     case Event.Type.FIREHOSE:
-      // TODO
+      const firehose = new FirehoseQueue();
```

Comment:
I agree. Also, simply we can use plain functions. However, I have concerns about performance. Since this file is the entry point it should be designed to work as fast as possible. Unlike the other parts, performance matters here. That is why we use terser to eliminate dead code and make other optimizations.

If we use the map to accomplish the same behavior, the javascript engine will take some evaluate the map and the functions inside. Also, I think we'll be fine as long as we keep things where they belong. Maybe in the future, we can introduce separate entry points for every enqueuer/queue type but for now, I think we'll be better if we leave it as is.

Reply:
SGTM :+1: